Tasks/Common/webdeployment-common-v2/xdttransformationutility.ts

```diff
@@ -31,12 +31,7 @@ export function applyXdtTransformation(sourceFile: string, transformFile: string
         "i"
     ];
     
-    var debugModeEnabled = tl.getVariable('system.debug');
-    if(debugModeEnabled && debugModeEnabled.toLowerCase() == 'true') {
-        cttArgsArray.push("verbose");
-        tl.debug('Enabled debug mode for ctt.exe');
-    }
-
+    cttArgsArray.push("verbose");
```

Comment:
I'm assuming, this will only effect the logs generated by ctt.exe and the task side logging verbosity will remain un-changed.  Is that right?

Reply:
Yes